src/ControllerV2/HomeController.php

```diff
@@ -19,4 +20,15 @@ public function renderBeneficiaryNav(?Beneficiaire $beneficiary): Response
 
         return $this->render('void.html.twig');
     }
+
+    #[IsGranted('ROLE_BENEFICIAIRE')]
+    #[Route(path: '/beneficiary/', name: 'beneficiary_home', methods: ['GET'])]
+    public function home(): Response
+    {
+        if ($subjectBeneficiary = $this->getUser()?->getSubject()) {
+            return $this->render('v2/beneficiary/home/home.html.twig', ['beneficiary' => $subjectBeneficiary]);
+        }
+
+        return $this->render('void.html.twig');
+    }
```

Comment:
Do you plan to add smoke test ?
Plus, users with 'ROLE_MEMBRE' can access this route, you will have an error